cmd/cni-metrics-helper/README.md

```diff
@@ -15,24 +15,29 @@ The following diagram shows how `cni-metrics-helper` works in a cluster:
 As you can see in the diagram, the `cni-metrics-helper` connects to the API Server over https (`tcp/443`), and another connection is created from the API Server to the worker node over http (`tcp/61678`). If you deploy Amazon EKS with the recommended security groups from [Restricting cluster traffic](https://docs.aws.amazon.com/eks/latest/userguide/sec-group-reqs.html#security-group-restricting-cluster-traffic), then make sure that a security group is in place that allows the inbound connection from the API Server to the worker nodes over `tcp/61678`.
 
 Adding the CNI metrics helper will publish the following metrics to CloudWatch:
-```
-"addReqCount",
-"assignIPAddresses",
-"awsAPIErr",
-"awsAPILatency",
-"awsUtilErr",
-"delReqCount",
-"eniAllocated",
-"eniMaxAvailable",
-"ipamdActionInProgress",
-"ipamdErr",
-"maxIPAddresses",
-"podENIErr",
-"reconcileCount",
-"totalIPAddresses",
-"totalIPv4Prefixes",
-"totalAssignedIPv4sPerCidr"
-```
+
+| Metric | Description |
+| ------ | ----------- |
+| addReqCount | Aggregate of the number of CNI ADD requests that require an IP address |
+| assignIPAddresses | Aggregate of the number of IP addresses assigned to pods |
```

Comment:
I'll adjust the wording, and I'm also thinking it might help reading to do away with specifying that a metric is an aggregate, and instead just add another column so the chart may be read like...

| Metric | Description | Representation |
| ------ | ------------ | --------------- |
| fooMetric | ... | Aggregate across all nodes |
| barMetric | ... | Maximum datapoint across all nodes | 

Something like this, what do you think?

Reply:
I like that, that's a great call! Instead of "Representation", maybe we can just have "Summation/Maximum", since that is all we support?